pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java

```diff
@@ -945,12 +955,35 @@ public void completeHandleFuture() {
     }
 
     public void exceptionHandleFuture(Throwable t) {
+        if(isRetryableException(t)) {
+            long retryTime = backoff.next();
+            log.warn("[{}][{}] Failed to init transaction pending ack handler. It will be retried in {} Ms",
+                    persistentSubscription.getTopic().getName(), subName, retryTime, t);
+            transactionOpTimer.newTimeout((timeout) -> init(), retryTime, TimeUnit.MILLISECONDS);
+            return;
+        }
+        log.error("PendingAckHandleImpl init fail! TopicName : {}, SubName: {}", topicName, subName, t);
```

Comment:
Please follow the same [format](https://github.com/apache/pulsar/pull/21274/files#diff-26056e00d43a29a9ff02a53654d692aef03e971fcec6c4f57b546afcfff2250dR960) instead of arbitrarily define the log format.